<?xml version="1.0" encoding="UTF-8"?>
<testsuite xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://maven.apache.org/surefire/maven-surefire-plugin/xsd/surefire-test-report.xsd" name="models.EndangeredAnimalsTest" time="12.308" tests="11" errors="11" skipped="0" failures="0">
  <properties>
    <property name="java.runtime.name" value="OpenJDK Runtime Environment"/>
    <property name="java.vm.version" value="11.0.6+10"/>
    <property name="sun.boot.library.path" value="/home/linus/.sdkman/candidates/java/11.0.6.hs-adpt/lib"/>
    <property name="maven.multiModuleProjectDirectory" value="/home/linus/Desktop/WildLifeTrackingApp"/>
    <property name="java.vm.vendor" value="AdoptOpenJDK"/>
    <property name="java.vendor.url" value="https://adoptopenjdk.net/"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="path.separator" value=":"/>
    <property name="java.vm.name" value="OpenJDK 64-Bit Server VM"/>
    <property name="sun.os.patch.level" value="unknown"/>
    <property name="user.country" value="US"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="/home/linus/Desktop/WildLifeTrackingApp"/>
    <property name="java.vm.compressedOopsMode" value="32-bit"/>
    <property name="java.runtime.version" value="11.0.6+10"/>
    <property name="java.awt.graphicsenv" value="sun.awt.X11GraphicsEnvironment"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.io.tmpdir" value="/tmp"/>
    <property name="line.separator" value="&#10;"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="os.name" value="Linux"/>
    <property name="classworlds.conf" value="/opt/maven/bin/m2.conf"/>
    <property name="sun.jnu.encoding" value="UTF-8"/>
    <property name="java.library.path" value="/usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib"/>
    <property name="maven.conf" value="/opt/maven/conf"/>
    <property name="jdk.debug" value="release"/>
    <property name="java.class.version" value="55.0"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="4.15.0-99-generic"/>
    <property name="library.jansi.path" value="/opt/maven/lib/jansi-native"/>
    <property name="user.home" value="/home/linus"/>
    <property name="user.timezone" value="Africa/Nairobi"/>
    <property name="java.awt.printerjob" value="sun.print.PSPrinterJob"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="java.specification.version" value="11"/>
    <property name="user.name" value="linus"/>
    <property name="java.class.path" value="/opt/maven/boot/plexus-classworlds-2.6.0.jar"/>
    <property name="java.vm.specification.version" value="11"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher heroku:deploy"/>
    <property name="java.home" value="/home/linus/.sdkman/candidates/java/11.0.6.hs-adpt"/>
    <property name="user.language" value="en"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="awt.toolkit" value="sun.awt.X11.XToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="11.0.6"/>
    <property name="securerandom.source" value="file:/dev/./urandom"/>
    <property name="java.vendor" value="AdoptOpenJDK"/>
    <property name="maven.home" value="/opt/maven"/>
    <property name="file.separator" value="/"/>
    <property name="java.version.date" value="2020-01-14"/>
    <property name="java.vendor.url.bug" value="https://github.com/AdoptOpenJDK/openjdk-support/issues"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="java.vendor.version" value="AdoptOpenJDK"/>
    <property name="sun.desktop" value="gnome"/>
    <property name="sun.cpu.isalist" value=""/>
  </properties>
  <testcase name="update_updatesAnimalDetails" classname="models.EndangeredAnimalsTest" time="1.566">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;41.84.152.6&quot;, user &quot;ubehwyfjkxozzy&quot;, database &quot;d8jlcrqdig8sdv&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
</error>
  </testcase>
  <testcase name="endangered_instantiatesCorrectly_true" classname="models.EndangeredAnimalsTest" time="0.599">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;41.84.152.6&quot;, user &quot;ubehwyfjkxozzy&quot;, database &quot;d8jlcrqdig8sdv&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
</error>
  </testcase>
  <testcase name="removeById_removesAnimalById" classname="models.EndangeredAnimalsTest" time="1.331">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;41.84.152.6&quot;, user &quot;ubehwyfjkxozzy&quot;, database &quot;d8jlcrqdig8sdv&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
</error>
  </testcase>
  <testcase name="removeAll_clearsAnimalDatabase" classname="models.EndangeredAnimalsTest" time="1.433">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;41.84.152.6&quot;, user &quot;ubehwyfjkxozzy&quot;, database &quot;d8jlcrqdig8sdv&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
</error>
  </testcase>
  <testcase name="save_savesAnimalToTheDatabase" classname="models.EndangeredAnimalsTest" time="1.309">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;41.84.152.6&quot;, user &quot;ubehwyfjkxozzy&quot;, database &quot;d8jlcrqdig8sdv&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
</error>
  </testcase>
  <testcase name="getAll_returnsAllInstancesOfAnimal" classname="models.EndangeredAnimalsTest" time="1.352">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;41.84.152.6&quot;, user &quot;ubehwyfjkxozzy&quot;, database &quot;d8jlcrqdig8sdv&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
</error>
  </testcase>
  <testcase name="save_assignsIdToAnimal" classname="models.EndangeredAnimalsTest" time="1.33">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;41.84.152.6&quot;, user &quot;ubehwyfjkxozzy&quot;, database &quot;d8jlcrqdig8sdv&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
</error>
  </testcase>
  <testcase name="endangered_instantiatesWithName_String" classname="models.EndangeredAnimalsTest" time="0.614">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;41.84.152.6&quot;, user &quot;ubehwyfjkxozzy&quot;, database &quot;d8jlcrqdig8sdv&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
</error>
  </testcase>
  <testcase name="find_returnsAnimalWithSameId_secondAnimal" classname="models.EndangeredAnimalsTest" time="1.33">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;41.84.152.6&quot;, user &quot;ubehwyfjkxozzy&quot;, database &quot;d8jlcrqdig8sdv&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
</error>
  </testcase>
  <testcase name="equals_returnsTrueIfNameAndRangerIdAreSame_true" classname="models.EndangeredAnimalsTest" time="0.716">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;41.84.152.6&quot;, user &quot;ubehwyfjkxozzy&quot;, database &quot;d8jlcrqdig8sdv&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
</error>
  </testcase>
  <testcase name="endangered_instantiatesWithRangerId_int" classname="models.EndangeredAnimalsTest" time="0.615">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;41.84.152.6&quot;, user &quot;ubehwyfjkxozzy&quot;, database &quot;d8jlcrqdig8sdv&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "ubehwyfjkxozzy", database "d8jlcrqdig8sdv", SSL off
</error>
  </testcase>
</testsuite>